feat(npm): adding `npmIt` alias (#10890)

denis-rossati opened the pull request:
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

- Added a alias to run `npm install-force`. This command run `npm install` followed immediately by an npm test as it says in [the docs](https://docs.npmjs.com/cli/v7/commands/npm-install-test)

- Also, I removed trailing spaces.

---

This PR closes the issue #10890

jmeridth reviewed (approved):
Why is the entire README changing and not just your new lines?

Looks good otherwise.

denis-rossati commented:
@jmeridth The changes in the table is because the 5th line has two white spaces after " `PATH="$(npm bin)":"$PATH"`".When I removed it, all other pipes were out of square, then I just aligned them. 

And in the `# Run npm publish` I just removed a white space.

You think this deserve a brand new PR? I could remove these changes and open an issue and the PR

jmeridth commented:
@denis-rossati all good.  thanks for explaining.

denis-rossati commented:
@mcornella Could you give your opinion? I think this is it for this alias
